plugins/json/src/main/java/org/apache/struts2/json/JSONUtil.java

```diff
@@ -53,7 +68,22 @@
      * @throws JSONException
      */
     public static String serialize(Object object) throws JSONException {
```

Comment:
It is currently used in the test cases
JSONResultTest - testJSONUtilNPEOnNullMethod()
JSONUtilTest = testSerializeDeserialize()
Thoughts?